osf_pigeon/settings/defaults.py

```diff
@@ -14,6 +14,7 @@
 DOI_FORMAT = os.environ.get("DOI_FORMAT")
 OSF_COLLECTION_NAME = os.environ.get("OSF_COLLECTION_NAME")
 ID_VERSION = os.environ.get("ID_VERSION")
+NUMBER_OF_WORKERS = os.environ.get('NUMBER_OF_WORKERS')
```

Comment:
Minor -- technically imprecise; `1` is the minimum possible number of workers (default varies, but min 5) and this is the `MAX_WORKERS`, actual number will vary by busyness.

Also, this should probably have a default.